ports/crunch/portfile.cmake

```
@@ -0,0 +1,34 @@
+vcpkg_check_linkage(ONLY_STATIC_LIBRARY)
+
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO rouault/crunch
+    REF 079b071dfc24d309fb22fa41ccd94a0a156cdb52
+    SHA512   486f1a3d25c777d93027579ded5f6838d3d056c020a9e4536cc669d78f999794ebab1fb311be575cc816e9918a9e8d4f4e7c80fea30329667401780ad7f96775
+    HEAD_REF build_fixes
+)
+
+vcpkg_configure_cmake(
```

```suggestion
vcpkg_cmake_configure(
```
For new ports, we prefer to use the new function instead of the deprecated one. Please see https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/vcpkg_configure_cmake.md.